src/runtime/database.h

```diff
@@ -169,6 +178,8 @@ struct Database {
   std::vector<JobEdge> get_edges();
   std::vector<JobTag> get_tags();
 
+  std::vector<RunReflection> get_runs() const;
```

Comment:
... and here you've dropped the markers entirely.  Looking at the surroundings, I think this one is correct, but the other two could do with standardization in their own ways.